src/wrapper/objects/mod.rs

```diff
@@ -36,6 +36,10 @@ pub use self::jbytebuffer::*;
 mod global_ref;
 pub use self::global_ref::*;
 
+// For when you want to store a reference to a java object
```

Comment:
"For storing a reference to a java object" if you want the same comment style as below. I'm not sure if it is needed at all.